src/en/help-oracle.md

```diff
@@ -2,94 +2,94 @@ Title: Using the Oracle cloud
 
 # Using the Oracle public cloud
 
-Juju has built-in support for [Oracle's Compute service][compute]. However, the
-following few steps are required before Juju can bootstrap Oracle's Compute
-cloud:
+Juju has built-in support for [Oracle Compute][oracle-compute], Oracle's public
+cloud. This means that there is no need to add the Oracle cloud to Juju. An
+exception to this are Oracle Compute trial accounts. Both types of accounts,
+paid and trial, are covered here.
 
-1. Associate Oracle's Ubuntu images with your Compute service. Juju uses these
-   for deployment.
+!!! Warning:
+    Support for Oracle Compute is only available via the [juju 2.2 dev
+    release][jujubeta] and is therefore not yet recommended for production use.
 
-1. For trial accounts, add the Oracle cloud to Juju. Endpoint details may need 
-   to be retrieved from the Oracle Compute *Dashboard*.
-   Juju already knows about endpoints for paid accounts so this step is only
-   necessary if using a trial account.
+The email you received upon signing up for Oracle Compute contains vital
+information you will need to complete the instructions presented here. Look
+for:
 
-1. Add authentication credentials for the specific Compute service to Juju.
+- 'My Services URL'
+- 'Identity domain'
+- 'My Account URL'
+- 'Username' (your email address)
+- 'Password' (temporary)
```

Comment:
This list is meant for information gathering. The bits are referred to later. Do I understand you correctly that you tried to put these bits together straight away?